I/ITK/build_tarballs.jl

```diff
@@ -0,0 +1,144 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITK"
+version = v"5.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://github.com/InsightSoftwareConsortium/ITK/releases/download/v$(version)/InsightToolkit-$(version).tar.gz", "57a4471133dc8f76bde3d6eb45285c440bd40d113428884a1487472b7b71e383")
```

Comment:
If this is a git-repo and what's being downloaded is the source code, then this should be a [`GitSource`](https://docs.binarybuilder.org/dev/reference/#BinaryBuilderBase.GitSource)

Reply:
Its an archive from a Github repository of ITK, i originally was following the pattern from other tarballs, seems like the latter is more appropriate.

Reply:
Since its a tar file, the docs refer to wrap it in an ArchiveSource function.

Reply:
Gotcha.  I'd still recommend a GitSource for this - just because it's a lot easier to update!

Reply:
Yes, making this a GitSource is better, because the hashes of these tarballs is actually not considered a stable property, and we have seen them accidentally get changed in the past: https://github.blog/2023-02-21-update-on-the-future-stability-of-source-code-archives-and-hashes/

Using the GitSource helps precent that because the commit hash we build from should be stable.